packages/monaco/src/browser/monaco-theming-service.ts

```diff
@@ -193,10 +193,11 @@ export class MonacoThemingService {
         return str;
     }
 
-    private cleanEmpty(obj: any): void {
+    /** removes all invalid theming values */
+    private clean(obj: any): void {
         for (const key in obj) {
             // eslint-disable-next-line no-null/no-null
-            if ([null, undefined].includes(obj[key])) {
+            if ([null, undefined].includes(obj[key]) || typeof obj[key] !== 'string') {
```

Comment:
The new check should cover the `null` and `undefined` cases as well, so might as well use it exclusively?